apps/services/regulations-admin-backend/src/app/app.module.ts

```diff
@@ -24,7 +25,11 @@ import { SequelizeConfigService } from './sequelizeConfig.service'
     DraftRegulationCancelModule,
     ConfigModule.forRoot({
       isGlobal: true,
-      load: [RegulationsClientConfig],
+      load: [
+        RegulationsClientConfig,
+        XRoadConfig,
+        NationalRegistryClientConfig,
+      ],
```

Comment:
I think you also need IdsClientConfig and an ids client with access to the national registry individuals scope.

Need to request a new machine client in all environments called `@island.is/clients/regulations-admin`, configure it in the infra file and add the client secret to all environments. 😅

Reply:
Great. So this should be taken care of now I believe (thanks 🙌).

Regarding the ids client with access to national registry scope that you mention. 
Am I not doing that now in the module for the service that uses the national-registry-v2 service? ([draft_regulation.module](https://github.com/island-is/island.is/pull/8983/files#diff-37935d2999a6f769ec294c3ad33f795f9c0fd1bb07ec7329e47044b0c62cf706R19))
Or maybe I'm misunderstanding a bit.